projects/storefrontstyles/scss/components/product/carousel/_product-carousel.scss

```diff
@@ -8,11 +8,10 @@
     padding-bottom: 10px;
     color: var(--cx-color-text);
     outline: none;
-    h4 {
-      font-size: 0.9rem;
+    h3 {
+      @include type('8');
```

Comment:
Is this right?

Reply:
Yes, it's using an existing type mixin instead of hardcoding font size. ideally, we don't want any hardcoded font values as it would lead to inconsistency and/or additional refactoring when we work with multiple themes.

Reply:
I understand that. But it doesn't make sense due to `type` values for h2 and h4...

Reply:
We are not changing any styling here. Just retain the existing style and reusing the mixin. there shouldn't be any visual change.

Reply:
spoke with Gil offline and resolved the conversation.